md/dates-management-tutorial.md

```diff
@@ -0,0 +1,67 @@
+# How to work with dates in forms and BDM
+
+Currently (until versions 7.4.x), only date selection is supported natively in the UI Designer date picker widget. So this page will only focus on date management (not date and time).
+
+## Define your BDM data
+
+Start by defining your business data model.
+
+It is recommended that you define the date attributes of your business data Objects with a **LONG** type instead of DATE if you want the value stored in the database to reflect the date entered by the user. Otherwise the timezone of the server will be used when converting the java date in an SQL date and, unless your server is configured in UTC, an offset will be applied. This will not be an issue if you do not query the business data database directly and only use the API (as the same conversion will be performed the other way around when retrieving the value from the database). In this case, you can safely use a DATE attribute type.
```

Comment:
We don't know yet for 7.5. But I can try with String to see how it goes.
The advantage of long compared to String is that there is no need to provide a validator in the contract to make sure that the date really exists. But this is true that if we move to String in 7.5, it may not be a good idea to advise Long...
I will try to implement the same process with Strings to see if it is doable before 7.5